app/src/main/res/layout/fragment_mentorship_tasks.xml

```diff
@@ -29,21 +28,24 @@
                 android:id="@+id/rvTasks"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_marginLeft="16dp"
-                app:layout_constraintTop_toBottomOf="@id/tvTodo"
-                app:layout_constraintVertical_weight="0.5" />
+                android:layout_marginStart="16dp"
+                android:layout_marginEnd="16dp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/tvTodo" />
 
             <androidx.appcompat.widget.AppCompatTextView
                 android:id="@+id/tvNoTask"
-                android:layout_width="match_parent"
+                android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_marginTop="12dp"
-                android:layout_marginEnd="12dp"
-                android:layout_marginBottom="12dp"
+                android:layout_marginStart="16dp"
```

Comment:
same goes for here, and for other lines

Reply:
Thanks for the review, I just updated the layout to use values from dimens.xml